packages/web-components/src/components/footer/footer-composite.ts

```diff
@@ -25,6 +25,7 @@ import './locale-button';
 import './legal-nav';
 import './legal-nav-item';
 import './legal-nav-cookie-preferences-placeholder';
+import '../locale-modal/locale-modal-container';
```

Comment:
Just a little suggestion from [the root cause of the bug](https://github.com/carbon-design-system/ibm-dotcom-library/blob/e744372f6107679fec71fa8a83439f7953eae144/packages/web-components/src/components/footer/footer-composite.ts#L180-L187). Note: `<dds-footer-container>`'s Redux store integration code covers everything for locale modal.

```suggestion
import '../locale-modal/locale-modal-composite';
```